### PR TITLE
Update client go to 3.0.0

### PR DESCRIPTION
**Pull request by surajssd**
Fixes https://github.com/kedgeproject/kedge/issues/81

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -6,10 +6,11 @@ import:
...
 - package: github.com/pkg/errors
 - package: k8s.io/client-go
-  version: ^2.0.0
+  version: 3.0.0
   subpackages:
   - pkg/api/v1
-  - pkg/runtime
   - pkg/apis
+- package: k8s.io/apimachinery
+  version: release-1.6
```
Comment (containscafeine) [inline glide.yaml]: Why did we pin the version to "release-1.6"? client-go v3.0.0 does that already, right? Any reason we had to specify that explicitly here?

Comment (surajssd) [inline glide.yaml]: If I don't pin it, it's master was being pulled leading it to not compile.

**Comment by surajssd** [comment]
I will wait on @kadel to merge this one, cause this is big one!

File: glide.yaml (hunk 1 above)
Comment (containscafeine) [inline glide.yaml]: gotcha, can you add that as a comment?

Comment (surajssd) [inline glide.yaml]: comment where?

Comment (containscafeine) [inline glide.yaml]: :) In glide.yaml, on the line above `- package: k8s.io/apimachinery`

Comment (surajssd) [inline glide.yaml]: is there a way to add this comment and then include this change in the commit which was added first?

Comment (kadel) [inline glide.yaml]: yep, add it as new commit, than run `git rebase -i HEAD~3`

Comment (containscafeine) [inline glide.yaml]: Sure, add the comment, create a new commit, and do a `git rebase -i HEAD~3` and then move the new commit up and squash with the previous one.

Comment (kadel) [inline glide.yaml]: but comments in glide.yml are bit tricky as everytime this file gets modified using `glide` command comments are lost :-( we must be careful with that

Comment (containscafeine) [inline glide.yaml]: haha, same time @kadel :P

Comment (containscafeine) [inline glide.yaml]: umm, @kadel `glide update` will modify glide.lock, not glide.yml, right?

Comment (surajssd) [inline glide.yaml]: done, thanks @kadel @containscafeine

Comment (kadel) [inline glide.yaml]: > umm, @kadel glide update will modify glide.lock, not glide.yml, right?

yes, only think that automatically modifies `glide.yaml` as far as I know is `glide get`

**Comment by surajssd** [comment]
@kadel is this good to go in?

**Comment by kadel** [comment]
> @kadel is this good to go in?

Haven't got time to look at it closely yet :disappointed:

File: pkg/transform/fixtures/single_container_object.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2017 The Kedge Authors All rights reserved.
```
Comment (kadel) [inline pkg/transform/fixtures/single_container_object.go]: Can you rabse with current master? 
This diff looks weird. `single_container_object.go` file shouldn't even exist after #180 being merged

Comment (surajssd) [inline pkg/transform/fixtures/single_container_object.go]: I did somehow it came, but yeah updated and pushed!

**Pull request merged**